actiontext/lib/generators/action_text/install/install_generator.rb

```diff
@@ -47,6 +47,9 @@ def create_actiontext_files
 
         copy_file "#{GEM_ROOT}/app/views/active_storage/blobs/_blob.html.erb",
           "app/views/active_storage/blobs/_blob.html.erb"
+
+        copy_file "#{GEM_ROOT}/app/views/layouts/action_text/contents/_content.html.erb",
+          "app/views/layouts/action_text/contents/_content.html.erb"
```

Comment:
@georgeclaghorn I've pushed up a change to copy over the layout as part of `rails generate action_text:install`.